baseplate/sidecars/event_publisher.py

```diff
@@ -215,6 +240,7 @@ def publish_events() -> None:
     publisher = BatchPublisher(metrics_client, cfg)
 
     while True:
+        sleep(0)
```

Comment:
It is the gevent sleep, but I can switch the import from `from gevent import sleep` to just `import gevent` so it's more obvious inline. And yep, I can move to the end.

Reply:
Oh actually, I noticed that since the while loop contains a `continue`, if we put the `sleep` at the end of the loop then it is harder to predict when it will actually allow the other routine to happen.